application/common/models/Reset.php

```diff
@@ -290,14 +308,8 @@ public function sendPhone()
 
         // Update db with code and increased attempts count
```

Comment:
The second half of this comment is now being done before this function is called.